content/ytt/docs/latest/lang-ref-ytt-overlay.md

```diff
@@ -408,6 +408,9 @@ _(this annotation has no parameters.)_
 
 **Note:** This action implies an `@overlay/match` selecting the last node. Any other `@overlay/match` annotation is ignored. 
 
+**History:**
+- v0.32.0 — append is the default array operation
```

Comment:
append isnt default operation. merge with non-matching behaviour that results in appending the item is default behaviour. not sure how to phrase that better.

Reply:
I like how this makes "default behavior" conceptually simpler — it's the same for everything: merge. It's just that when a merge fails to match, the fallback is an append (like an "upsert").

👍🏼